mcs/class/corlib/corlib_xtest.dll.sources

```diff
@@ -18,7 +18,7 @@
 
 ../../../external/corefx/src/System.Reflection/tests/Common.cs
 
-../../../external/corefx/src/System.Reflection/tests/*.cs:AssemblyTests.cs,AssemblyNameTests.cs,MethodInfoTests.cs,ModuleTests.cs
+../../../external/corefx/src/System.Reflection/tests/*.cs:AssemblyTests.cs,TypeInfoTests.netcoreapp.cs,AssemblyTests.netcoreapp.cs,AssemblyNameTests.cs,MethodInfoTests.cs,ModuleTests.cs
```

Comment:
Why is this change needed?

Reply:
@marek-safar "temp" fix for the changes I pulled with "bump corefx". Will get rid of it once Maxim merges his PR